crates/bpf-common/include/output.bpf.h

```diff
@@ -47,6 +47,18 @@ struct {
   __uint(max_entries, 1);
 } temp_map SEC(".maps");
 
+// The status map contains configuration status for the eBPF program:
+// - The STATUS_INITIALIZED entry indicates the perf event array initialization
+//   status. On startup it's set to 0, when the userspace program opens the perf
+//   event array it's set to 1. The eBPF program emits events only when it's 1.
+#define STATUS_INITIALIZED 0
+struct {
+  __uint(type, BPF_MAP_TYPE_ARRAY);
+  __type(key, u32);
+  __type(value, u32);
+  __uint(max_entries, 1);
+} status_map SEC(".maps");
```

Comment:
I would call "ready map", " init map" or something siminal, because it's set one time, and not a full status

Reply:
Ok. I was planning to add more stuff to it, which would have made the `status` name more fitting, but we can always rename it later.